setup.sh

```
@@ -0,0 +1,34 @@
+  cp app/src/main/cpp/native-lib.cpp.example app/src/main/cpp/native-lib.cpp
+  echo "✅ native-lib.cpp created!"
+fi
+
+# Step 3 - Copy dummy google-services.json
+echo "📄 Copying dummy google-services.json..."
+cp dummy/google-services.json app/src/sakshamUat/google-services.json
+echo "✅ google-services.json copied!"
+
+# Step 4 - Load .env variables
+echo "📦 Loading environment variables..."
+export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)